source/parser.hera

```diff
@@ -4147,6 +4148,7 @@ Reset
       set(b) {
         if (b) {
           this.autoVar = true
+          this.autoLet = false
```

Comment:
This isn't necessary here or in the else branch below since `autoLet` is unrelated to the `coffeeCompat` flags.

Reply:
`coffeeCompat` implies `autoVar`, right?  How do `autoVar` and `autoLet` interact?  Presumably one should disable the other, maybe whichever is specified last.  So you could `"civet coffeeCompat autoLet"` and `autoLet` would win.

Reply:
To resolve that we should check and process `autoLet` before `autoVar` inside `processProgram`.